doc/source/cluster/kubernetes/getting-started/rayjob-quick-start.md

```diff
@@ -191,6 +217,24 @@ kubectl get raycluster
 
 ## Step 10: Clean up
 
+::::{tab-set}
+
+:::{tab-item} Arm64 (Apple Silicon)
+
+```sh
+# Step 10.1: Delete the RayJob
+curl -s https://raw.githubusercontent.com/ray-project/kuberay/v1.2.2/ray-operator/config/samples/ray-job.shutdown.yaml | sed 's/2.9.0/2.9.0-aarch64/g' | kubectl delete -f -
+
+# Step 10.2: Delete the KubeRay operator
+helm uninstall kuberay-operator
+
+# Step 10.3: Delete the Kubernetes cluster
+kind delete cluster
+```
+:::
+
+:::{tab-item} X86-64 (Intel/Linux)
+
```

Comment:
We don't care the image when executing `kubectl delete`

Reply:
ok , I will remove the tab_set in this section